src/lib.rs

```diff
@@ -169,6 +171,18 @@ impl Blockchain {
         let tx = psbt.internal.lock().unwrap().clone().extract_tx();
         self.get_blockchain().broadcast(&tx)
     }
+
+    fn get_height(&self) -> Result<u32, Error> {
+        return self.get_blockchain().get_height();
```

Comment:
To return the result I think you need to remove the trailing `;`, otherwise the returned type will be the empty type `()`. You should create a little test to confirm a new Blockchain returns `Ok(0)` for this function to confirm.
```suggestion
        return self.get_blockchain().get_height()
```

Reply:
I may be wrong about needing to remove the `;` since this seems to compile fine, if it wasn't returning a Result the CI should be throwing an error.

Reply:
My understanding is that the last line is an expression if it doesn't have the semicolon (and is automatically the return value of a function). Moreover I think that's the more idiomatic way to write Rust, but for some reason I had problems with it when I wrote it earlier, yet now it compiles fine. So I'll go with that!

```rust
    fn get_height(&self) -> Result<u32, Error> {
        self.get_blockchain().get_height()
    }
````